clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v1/api/KubernetesClientApiAdapter.groovy

```diff
@@ -141,33 +142,41 @@ class KubernetesClientApiAdapter {
 
   List<V1beta1StatefulSet> getStatefulSets(String namespace) {
     exceptionWrapper("statefulSets.list", "Get Stateful Sets", namespace) {
-      /*
-       "fixme" and note this is k8s-client api issue and we are working this as workaround.
+      try {
+        /*
+          FIXME: note this is k8s-client api issue and we are working this as workaround.
```

Comment:
Why is this a client issue? If your k8s server doesn't support this endpoint it fails like it's expected to

Reply:
This is for hard coded "kind" value.